modules/vstudio/tests/vc2010/test_globals.lua

```diff
@@ -277,6 +277,25 @@
 		]]
 	end
 
+	
+	function suite.windowsTargetPlatformVersionLatest_on2019()
+		p.action.set("vs2019")
+		systemversion "latest"
+		local oldRegistry = os["getWindowsRegistry"]
+		os["getWindowsRegistry"] = function (key) return "10.0.11111" end
```

Comment:
Keeping this override makes the unit test confusing, it's probably best to remove this and make it clear here that VS2019 uses `10.0` to indicate "use latest SDK".

Reply:
Good point. Fixed!